docker-hub/orgs.md

```diff
@@ -9,151 +9,228 @@ redirect_from:
 Docker Hub organizations let you create teams so you can give your team access
 to shared image repositories.
 
-- **Organizations** are collections of teams and repositories that can be managed together.
-- **Teams** are groups of Docker Hub users that belong to an organization.
+An **Organization** is a collection of teams and repositories
+that can be managed together. Docker users cannot belong directly to an
+organization. They belong only to teams within an organization. When you first
+create an organization, you’ll see that you have a team, the owners team, with a
+single member. An organization owner can then create new teams and add members
+to an existing team using Docker IDs or email address and by selecting a team
+the user should be part of.
+
+A **Team** is a group of Docker users that belong to an organization. An
+organization can have multiple teams. When you first create an organization,
+you’ll see that you have a team, the owners team, with a single member. An
+organization owner can then create new teams and add members to an existing team
+using Docker IDs and by selecting a team the user should be part of.
+
+> **Note:**
+>
+> In Docker Hub, users cannot belong directly to an organization.
+They belong to teams within an organization. Organization owners (Admins) can
+[configure repository permissions](#configure-repository-permissions) on a per-team basis. For example, as an org owner,
+you can specify that all teams within an organization have Read and Write access
+to repositories A and B, whereas only specific teams have Admin access to those
+repositories. Org owners have full administrative access to all repositories
+within the organization.
+
+## Create an organization
```

Comment:
Keep in mind there are multiple ways to create orgs (e.g. convert your user account profile to an org).

Someone reading just this section might want to know there are other ways (at least that way).

I wouldn't deep dive here, but say maybe one sentence that orgs can be created brand new or converted from user accounts (and link to a different section on how to convert).

Reply:
Good point. I've added an intro with info on two ways of creating an org and added a link to the Convert a user account into a org section